doc/embedded/crosscompile_example.md

```diff
@@ -0,0 +1,132 @@
+## Crosscompile mlpack example for an embedded hardware
+
+In this article, we explore how to add mlpack to a CMake project that cross-compiles
+code to embedded hardware.  See also these related other guides, which may be useful
+to read before this one:
+ * [Run mlpack bindings on a Raspberry Pi](crosscompile_armv7.md)
+ * [Set up cross-compilation toolchain for mlpack](supported_boards.md)
+
+
+### Cloning mlpack example respository
+
+mlpack has an [example repository](https://github.com/mlpack/examples) that
+
+shows a number of applications and use cases for mlpack, including embedded
+deployment.  In this tutorial, we are interested in the `embedded/` directory,
+which provides a CMake project template that compiles a random forest
+application to embedded hardware.  This project template can be adapted to a new
+project, or its pieces can be incorporated into an existing CMake project.
+
+The first step is to clone the examples repository:
+
+```sh
+git clone git@github.com:mlpack/examples.git
+```
+
+Next, let's look at the `CMakeLists.txt` (e.g. the CMake configuration) in the
+`embedded/crosscompile_random_forest/` directory.
+
+### `CMakeLists.txt`
+
+The first part of the code, printed below or
+[available here](https://github.com/mlpack/examples/blob/master/embedded/crosscompile_random_forest/CMakeLists.txt),
```

Comment:
I guess we need to update that example with another PR?  It should be very simple I suppose.

Reply:
yes I can do it in another one and then come back to this one if necessary

Reply:
Yes